drivers/hud_path_report/spec/models/fy2024/question_eight_to_sixteen_spec.rb

```diff
@@ -0,0 +1,67 @@
+###
+# Copyright 2016 - 2023 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+require 'rails_helper'
+require_relative 'path_context'
+
+RSpec.describe HudPathReport::Generators::Fy2024::QuestionEightToSixteen, type: :model do
+  include_context 'path context FY2024'
+
+  before(:all) do
+    default_setup
+    run(default_filter, HudPathReport::Generators::Fy2024::QuestionEightToSixteen::QUESTION_NUMBER)
+  end
+
+  after(:all) do
+    cleanup
+  end
+
+  # DEPRECATED_FY2024 - these are expected to fail until we re-write the report
```

Comment:
Are these deprecations still true?

Reply:
I _think_ they should pass now, this was because race/ethnicity changed.

Reply:
Looks like the five commented-out tests still fail